pkgs/top-level/all-packages.nix

```diff
@@ -186,6 +186,10 @@ in
 
   corgi = callPackage ../development/tools/corgi { };
 
+  colobot = callPackage ../games/colobot {};
+
+  colobot-data = callPackage ../games/colobot/data.nix {};
```

Comment:
does this need to be exposed as a top-level attr?

Reply:
could just do something like:
```
let
  colobot-data = callPackage ./data.nix { };
in
...
```
from default.nix

Reply:
You mean the data?

Don't know if there is another pattern

Reply:
Yea, I meant the data

Reply:
Ok pushed an update